src/GraphODataTemplateWriter/CodeHelpers/TypeScript/TypeHelperTypeScript.cs

```diff
@@ -59,7 +59,15 @@ public static String UpperCaseFirstChar(this String s)
         {
             return char.ToUpper(s[0]) + s.Substring(1);
         }
-        
-        
+
+        public static string GetSanitizedLongDescription(this OdcmProperty property)
+        {
+            var description = property.LongDescription ?? property.Description;
+            if (description != null)
+            {
+                return description.Replace("<", "&lt;").Replace(">", "&gt;").Replace("&", "&amp;");
```

Comment:
I don't understand the need for the replacements in this. why do we need that.?

Reply:
@muthurathinam We've found these characters coming from the documentation in the past. These are not valid characters in  doc comments for .Net IntelliSense files IIRC. I couldn't find a list of invalid characters for JSDoc. If this isn't a concern for the typings, I can remove. Just let me know if there is any reason to keep this for JS and if so, which characters should we remove. I'm going to work under the assumption that they aren't valid and will checkin a change.

Reply:
@MIchaelMainer You are right, we should escape these characters. Having these (<, >, &) characters does not effects the editor intellisense but these characters are trimmed when we generate the html pages from these jsDoc comments and when we do this escaping the comments looks as we expected

![code](https://user-images.githubusercontent.com/6259786/49566262-103e0500-f950-11e8-8a65-79490e0c7017.PNG)


![jsdoc_gen](https://user-images.githubusercontent.com/6259786/49566289-29df4c80-f950-11e8-9a06-5512b4855bbb.PNG)